doc/man/8/ceph-volume.rst

```diff
@@ -39,6 +39,8 @@ Commands
 inventory
 ---------
 
+.. program:: ceph-volumes inventory
```

Comment:
is there a reason why we need to use "ceph-volumes" instead of "ceph-volume"?

Reply:
@neha-ojha no particular reason. it's a typo. fixed and repushed.

would you mind taking another look?